test_cases/structured_geocoding.json

```diff
@@ -1065,6 +1083,150 @@
           }
         ]
       }
+    },
+
+    {
+      "id": 1100,
+      "status": "pass",
+      "user": "trescube",
+      "type": "dev",
+      "notes": "query+address should ignore address",
```

Comment:
refresh my memory, why should it ignore the address? is that how we're faking "find police stationgs near 1090 N Charlotte St, Lancaster, PA" queries?